README.md

```diff
@@ -26,6 +26,16 @@ Wormhole Router keeps track of each Domain's Gateway and routes `requestMint()`
 
 ![FastWithdrawal](./docs/fw.png?raw=true)
 
+## Governance controlled parameters:
+
+* `vat.ilk[wormholeJoinIlk].line`- *global debt ceiling*. Usually should be a sum of all debt ceilings available for each domain.
```

Comment:
The word `global` might be confused with `vat.Line` which is the global debt ceiling of MCD. Maybe call it `all domains debt ceiling` or something similar

Reply:
LFW and I just discussed calling it the "Wormhole Debt Ceiling", and the individual ones can be "Domain Debt Ceiling" or something.

Reply:
Yeah just read it, was coming back to mention it :)

Reply:
`line`, `pine`, `tine`,  got it.